custom_components/localtuya/light.py

```diff
@@ -188,7 +188,9 @@ def hs_color(self):
         """Return the hs color value."""
         if self.is_color_mode:
             return self._hs
-        if self.supported_features & SUPPORT_COLOR:
+        if self.supported_features & SUPPORT_COLOR and not self.has_config(
```

Comment:
I think you can do this:

```python
if self.supported_features & (SUPPORT_COLOR | SUPPORT_COLOR_TEMP):
```

Reply:
It's a `not`,  I mean, in case SUPPORT_COLOR but does not SUPPORT_COLOR_TEMP

Reply:
Ah, right, missed that. But you should be able to use `not. self.supported_features & SUPPORT_COLOR_TEMP` for consistency.

Reply:
Ok